.github/workflows/release.yml

```diff
@@ -90,6 +90,7 @@ jobs:
       - uses: actions/upload-artifact@v4
         if: success()
         with:
+          name: linux
```

Comment:
which part is this fixing?

Reply:
If the name is not set, they all write to `artifact`. I think the last job wins. A customer pointed out to us that [v0.0.9](https://github.com/EngFlow/auth/releases/tag/v0.0.9) lacked a Windows artifact: this is why.

Reply:
Oh, I see this names the artifact.